uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/test/resources/META-INF/services/org.uberfire.java.nio.file.spi.FileSystemProvider

```diff
@@ -30,6 +30,6 @@
 # limitations under the License.
 #
 
-org.uberfire.java.nio.fs.file.SimpleFileSystemProvider  # file system provider, also default (1st)
 org.uberfire.java.nio.fs.jgit.JGitFileSystemProvider
+org.uberfire.java.nio.fs.file.SimpleFileSystemProvider  # file system provider, also default (1st)
```

Comment:
@adrielparedes It would be good if we could clean the comments from the ServiceLoader properties file a bit. Things like, ', also default (1st)' may cause confusions down the road. Thanks!

Reply:
I will do it but just before merge, because If I do it now we are going to lose the FDB, is that ok?